packages/slice-machine/lib/models/common/http/DefaultClient.ts

```diff
@@ -42,26 +47,28 @@ function createApiUrl(base: string, { STAGE, PROD }: ApiSettings): string {
   return PROD;
 }
 
+type Body = Slice<AsObject> | Record<string, unknown> | string;
```

Comment:
I'm guessing it was for the linter that you changed those stuff 👍 😄 
Isn't `Slice<AsObject>` already included in `Record<string, unknown>`, why do you need to specify for this type ?

Reply:
Apparently not, as I think it's an array of some sort

Reply:
hmmm ok, weird because everywhere it is used, the type use to be `object | string` which seems coherent.

Reply:
Yes object causes linter warnings